pulpcore/app/models/content.py

```diff
@@ -252,6 +252,15 @@ class Meta:
         verbose_name_plural = 'content'
         unique_together = ()
 
+    def __init__(self, *args, **kwargs):
+        """
+        Add a plugin name to the type of Content to distinguish between the same content types
+        across plugins.
+        """
+        super().__init__(*args, **kwargs)
+        self.type = '{app_label}.{type}'.format(app_label=self._meta.app_label,
+                                                type=self.TYPE)
```

Comment:
This affects the `type` which is stored in the DB in the `pulp_app_content`, so to search content by type one will need to specify plugin name and type of content in a specific format, e.g. `'pulp_file.file'`.

Does it make sense to add new field/column `app_label` to the MasterModel? At the moment the change above affects only the Content model. If `app_label` is added to the MasterModel, it will affect all current master models.